### DISCUSSION TITLE
Allow to change 'Null' data behavior in each chart

### DISCUSSION THREAD

**Discussion by kamalkeshavani-aiinside** [apache/superset#20138] [Ideas]
Different charts show NULL values in different ways:
- No Data No data after filtering or data is NULL of the latest time record
- null
- NaN

We would like to change this string shown in charts for NULL case to custom(user input) string.

**Comment by cccs-tom** [comment]
+1 We ran into this as well. Also, the Table viz displays a grayed-out `N/A` for null values in a column. At the very least, these should be consistent, but I agree that the optimal solution would be to allow the chart designer to specify the placeholder text in this scenario.

**Comment by amasidlovermedoc** [comment]
Totally agree with the above the 'n/a' needs to be configurable - the only way we can work around this is either to have a CRON job that runs on the data and replaces all NULLs in all the char/varchar/text/memo fields in all the source tables with empty strings...

**Comment by ampil** [comment]
I know it's old, but at least it may help someone.

In tables, N/A cells have a HTML class `dt-is-filter dt-is-null`.
You may Edit CSS of your dashboard and try this to 'hide' N/A values:

```
.dt-is-null {
  color: #FFFFFF !important;
}
```


Before:
![image](https://github.com/user-attachments/assets/fb376990-58cc-4c47-9bbb-afedc61e66a1)


After:
![image](https://github.com/user-attachments/assets/a24803ad-ec63-4a51-9db0-9e78e0172e9f)
